plugins/scaffolder-backend/src/scaffolder/stages/publish/bitbucket.test.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const mockResponse = jest.fn();
+jest.mock('./helpers');
+jest.mock('cross-fetch', () => mockResponse);
```

Comment:
You can use `msw` to mock out network requests instead of this way.\

Checkout the ADR and other examples in the repository. https://backstage.io/docs/architecture-decisions/adrs-adr007

Reply:
Thanks. The PR has been updated to use `msw` for mocking the Bitbucket APIs